### PR TITLE
Remove debug define statements

### PR DESCRIPTION
**Pull request by mafonsoN**
List of defines that have been removed:
#define DISABLE_QPSM_1PASS                  1
#define PME_UP_TO_4_REF                     1
#define TH_BASED_CFL                        1
#define TH_BASED_CHROMA_SEARCH              1
#define NSQ_HV                              1
#define OMARK_LAMBDA                        1
#define OMARK_HBD1_CDEF                     1
#define OMARK_HBD0_ED                       1
#define OMARK_HBD0_IFS                      1
#define OMARK_HBD0_MD                       1
#define OMARK_HBD0_RDOQ                     1
#define RDQO_ON_HBD0                        1
#define PICT_SWITCH                         1
#define QUANT_CLEANUP                       1
#define QUANT_HBD0_FIX                      1
#define NEW_MD_LAMBDA                       1
#define FEB19_PD0_TH                        1
#define TUNE_SUBPEL_SEARCH                  1
#define JAN6_PRESETS                        1
#define MRP1D_COMPOUND                      1
#define PALETTE_TL                          1
#define FEB27_ADOPTIONS                     1
#define FEB24_ADOPTIONS                     1
#define M0_FEB23_ADOPTIONS                  1
#define M2_FEB22_ADOPTION                   1
#define M0_FEB21_ADOPTIONS                  1
#define M1_FEB22_ADOPTIONS                  1
#define SC_FEB12_ADOPTION                   1
#define M2_FEB14_ADOPTION                   1
#define M0_FEB22_ADOPTIONS                  1
#define FEB14_ADOPTIONS                     1
#define SC_FEB4_ADOPTION                    1
#define M1_FEB4_ADOPTION                    1
#define SC_PRESETS_OPT                      1
#define PRESETS_OPT                         1
#define SC_ADOPTION_FEB_19                  1
#define FEB18_ADOPTIONS                     1
#define MR_MODE_CLEAN_UP                    1
#define M2_ADOPTIONS                        1
#define ENABLE_HME_360P                     1
#define MR_FEB4_ADOPTION                    1
#define SPATIAL_DOMAIN_ONLY_LAST_STAGE      1
#define MD_STAGE_1_CLASS_PRUNNING_TH        1
#define MD_STAGE_2_CAND_PRUNNING_TH         1
#define SC_REDUCE_DIFF                      1
#define JAN31_M2                            1
#define OPT_REC_ME                          1
#define OPT_REC_SUBP                        1
#define SKIP_DEPTH                          1
#define REVERT_TF_SEETINGS                  1
#define PLANE_WISE_TF                       1
#define PLANE_WISE_TF_OPT                   1
#define IMPROVED_TF_ME_INPUT                1
#define CHROMA_SEARCH_OPT                   1
#define INFR_OPT                            1
#define MOVE_OPT                            1
#define COMP_OPT                            1
#define NSQ_ME_OPT                          1
#define ALTREF_PACK                         1
#define NICS_CLEANUP                        1
#define COMP_SIMILAR                        1
#define INTRA_SIMILAR                       1
#define OIS_MEM                             1
#define SB_SIZE_ADOPTION                    1
#define MULTI_STAGE_ME                      1
#define MUS_ME                              1
#define SHUT_HME_L1_CHECK                   1
#define MUS_ME_FP                           1
#define MUS_ME_FP_SB                        1
#define SKIP_ME_BASED_ON_HME                1
#define REDUCE_ME_FOR_LOW_M_SB              1
#define SWITCHED_HALF_PEL_MODE              1
#define SC_HME_PRUNING                      1
#define MC_DYNAMIC_PAD                      1
#define DIST_BASED_ME_SEARCH_AREA           1
#define GLOBAL_WARPED_MOTION                1
#define PRED_STR_UPDATE                     1
#define MRP_31B_SUPPORT                     1
#define TILES_PARALLEL                      1
#define LOW_DELAY_TUNE                      1
#define ALTREF_IMPROVEMENT                  1
#define MD_RATE_EST_ENH                     1
#define WARP_IMPROVEMENT                    1
#define QPS_CHANGE                          1
#define QPS_CHANGE_II                       1
#define FASTER_RDOQ                         1
#define FP_QUANT_BOTH_INTRA_INTER           1
#define ENHANCED_SQ_WEIGHT                  1
#define ENHANCED_ME_MV                      1
#define TXS_DEPTH_2                         1
#define FEB28_ADOPTIONS                     1
#define PME_OFF_IF_NSQ                      0
#define COMP_OFF_IF_NSQ                     0
#define OBMC_OFF_IF_NSQ                     0
#define WARP_OFF_IF_NSQ                     0
#define MRP_OFF_IF_NSQ                      0
#define CFL_OFF_IF_NSQ                      0

### CODE REVIEW

File: Source/Lib/Encoder/Codec/EbEncDecProcess.c [hunk 1]
```diff
@@ -2262,1168 +2171,109 @@ EbErrorType signal_derivation_enc_dec_kernel_oq(
```
Comment (PhoenixWorthVCD) [inline Source/Lib/Encoder/Codec/EbEncDecProcess.c]: @hguermaz Should the cfl and nsq MD signals be removed?

Comment (hguermaz) [reply Source/Lib/Encoder/Codec/EbEncDecProcess.c]: @PhoenixWorthVCD just removed.

Comment (hguermaz) [reply Source/Lib/Encoder/Codec/EbEncDecProcess.c]: @PhoenixWorthVCD just removed.

Comment (hguermaz) [reply Source/Lib/Encoder/Codec/EbEncDecProcess.c]: @PhoenixWorthVCD just removed.

**Comment by hguermaz** [comment]
@mafonsoN 
2 more macros:
#define PD_REF_ADP_OTHERWISE 0
#define PD_BASE_ADP_OTHERWISE 0

**Comment by mafonsoN** [comment]
> @mafonsoN
> 2 more macros:
> #define PD_REF_ADP_OTHERWISE 0
> #define PD_BASE_ADP_OTHERWISE 0

Thanks @hguermaz, I'll add these.

**Comment by mafonsoN** [comment]
@ttrigui can you fix these warnings in the current cs2?

```
/Users/mafonso/repos/SVT-AV1-masterhash/Source/Lib/Common/ASM_AVX2/EbPictureOperators_Intrinsic_AVX2.c:1911:22: warning:
      comparison of integers of different signs: 'int32_t' (aka 'int') and 'uint32_t' (aka 'unsigned int') [-Wsign-compare]
            for (; k < width; k++) {
                   ~ ^ ~~~~~
/Users/mafonso/repos/SVT-AV1-masterhash/Source/Lib/Common/ASM_AVX2/EbPictureOperators_Intrinsic_AVX2.c:1937:18: warning:
      comparison of integers of different signs: 'int32_t' (aka 'int') and 'uint32_t' (aka 'unsigned int') [-Wsign-compare]
        for (; k < width; k++) {
               ~ ^ ~~~~~
```

```
/Users/mafonso/repos/SVT-AV1-masterhash/Source/Lib/Encoder/Codec/EbEncInterPrediction.c:2143:59: warning: unused parameter
      'bit_depth' [-Wunused-parameter]
static INLINE void build_obmc_inter_pred_left_hbd(uint8_t bit_depth, MacroBlockD *xd,
                                                          ^
```

**Pull request closed without merging**